src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -308,3 +310,96 @@ export const answerReviewLabel = (key, formValues) => {
     }
   }
 };
+
+export const determineBoardObj = (formResponses, noDRB) => {
+  if (!formResponses) return null;
+
+  const prevAppType = [
```

Comment:
Consider: looking at logic blocks like this, I'm not sure we're gaining a lot of readability as opposed to the original code. For example:

<img width="286" alt="Screenshot 2024-07-08 at 9 22 40 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/19175324/2dcf4a86-eb2f-4e0b-b21f-64e53f13841a">

Rather than using generic variables like `PREV_APPLICATION_TYPE_1`, should we use something like `PREV_APPLICATION_DOC_REVIEW` and `PREV_APPLICATION_PERSONAL_APP_REVIEW`? It helps give ore context for reference when using code blocks like this:

<img width="552" alt="Screenshot 2024-07-08 at 9 25 01 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/19175324/d9835c1f-0db3-4d5a-8b1a-52622e0c00af">

If you decide to make those updates, I wouldn't say they necessarily belong in this PR. We could always create a new PR just for that purpose. Food for thought.

Reply:
This makes sense, I can update consts where it makes sense to add more context to the responses.